modules/components/pages/inputs/pg_stream.adoc

```diff
@@ -0,0 +1,395 @@
+= pg_stream
+// tag::single-source[]
+:type: input
+:page-beta: true
+:categories: ["Services"]
+
+component_type_dropdown::[]
+
+Streams data changes from a PostgreSQL database using logical replication. There is also a configuration option to <<stream_snapshot,stream all existing data>> from the database.
+
+ifndef::env-cloud[]
+Introduced in version 4.40.0.
+endif::[]
+
+
+```yml
+# Configuration fields, showing default values
+input:
+  label: ""
+  pg_stream:
+    dsn: postgres://foouser:foopass@localhost:5432/foodb?sslmode=disable # No default (required)
+    batch_transactions: true
+    stream_snapshot: false
+    snapshot_memory_safety_factor: 1
+    snapshot_batch_size: 0
+    schema: public # No default (required)
+    tables: [] # No default (required)
+    checkpoint_limit: 1024
+    temporary_slot: false
+    slot_name: "" # No default (optional)
+    pg_standby_timeout: 10s
+    pg_wal_monitor_interval: 3s
+    max_parallel_snapshot_tables: 1
+    auto_replay_nacks: true
+    batching:
+      count: 0
+      byte_size: 0
+      period: "" # No default (optional)
+      check: "" # No default (optional)
+      processors: [] # No default (optional)
+```
+
+== Change data capture in the `pg_stream` input
+
+The `pg_stream` input uses logical replication to capture changes made to a PostgreSQL database in real time and streams them to Redpanda Connect. Redpanda Connect uses this replication method to allow you to choose which database tables in your source database to receive changes from.
+
+=== Prerequisites
+
+- PostgreSQL version 10 or later
+- Logical replication enabled
+
+=== Enable logical replication
+
+Before you run a pipeline using the `pg_stream` input, you must enable logical replication on your PostgreSQL source database.
+
+To check whether logical replication is already enabled, run the following command:
+
+```SQL
+SELECT name,setting FROM pg_settings WHERE name IN ('wal_level','rds.logical_replication');
+```
+If the `wal_level` value is `logical`, you can start to use this connector. Otherwise, choose from the following sets of instructions to update your replication settings.
+
+==== Self-Hosted PostgreSQL
+
+You must use an account with sufficient permissions to update your replication settings:
+
+. Open the `postgresql.conf` file.
+. Find the `wal_level` parameter and uncomment it.
+. Update the parameter value to `wal_level = logical`. If you already use replication slots, you may need to increase the limit on replication slots (`max_replication_slots`). The `max_wal_senders` parameter value must also be greater than or equal to `max_replication_slots`.
+. Restart the PostgreSQL server.
```

Comment:
Repliation connection settings are still to be added. I need some more context on these.

Reply:
Please check Self-Hosted PostgreSQL section for accuracy